src/M6Web/Bundle/LogBridgeBundle/DependencyInjection/M6WebLogBridgeExtension.php

```diff
@@ -27,26 +27,65 @@ public function load(array $configs, ContainerBuilder $container)
         $loader = new Loader\YamlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
         $loader->load('services.yml');
 
-        $container->setParameter('m6web_log_bridge.logger_service', $config['logger']);
+        $container->setParameter('m6web_log_bridge.logger.service', $config['logger']['service']);
+        $container->setParameter('m6web_log_bridge.logger.channel', $config['logger']['channel']);
         $container->setParameter('m6web_log_bridge.resources', $config['resources']);
         $container->setParameter('m6web_log_bridge.prefix_key', $config['prefix_key']);
         $container->setParameter('m6web_log_bridge.content_formatter', $config['content_formatter']);
 
         if (!empty($config['ignore_headers'])) {
             $container
-                    ->setParameter(
-                        'm6web_log_bridge.ignore_headers',
-                        array_merge(
-                            $container->getParameter('m6web_log_bridge.ignore_headers'),
-                            $config['ignore_headers']
-                        )
-                    );
+                ->setParameter(
+                    'm6web_log_bridge.ignore_headers',
+                    array_merge(
+                        $container->getParameter('m6web_log_bridge.ignore_headers'),
+                        $config['ignore_headers']
+                    )
+                );
         }
 
+        $this->loadLogger($container);
         $this->loadListener($container);
     }
 
     /**
+     * loadLogger
+     *
+     * @throws \Exception Logger service must be implement "Psr\Log\LoggerInterface"
+     * @throws \Exception Logger channel must be a valid monolog channel name (non empty string)
+     *
+     * @param ContainerBuilder $container Container
+     */
+    protected function loadLogger(ContainerBuilder $container)
+    {
+        $loggerServiceName  = $container->getParameter('m6web_log_bridge.logger.service');
+        $loggerChannel      = $container->getParameter('m6web_log_bridge.logger.channel');
+
+        if ($loggerServiceName !== self::DEFAULT_LOGGER_SERVICE_NAME) {
```

Comment:
And i dont understand why to do this test ... Why all check cant be done on default logger class ?